cookbook/profiler/profiling_data.rst

```diff
@@ -0,0 +1,60 @@
+.. index::
+   single: Profiling; Profiling data
+
+How to Access Profiling Data Programmatically
+=============================================
+
+Most of the times, the profiler information is accessed and analyzed using its
+web-based visualizer. However, you can also retrieve profiling information
+programmatically thanks to the methods provided by the ``profiler`` service.
+
+When the response object is available, use the
+:method:`Symfony\\Component\\HttpKernel\\Profiler\\Profiler::loadProfileFromResponse`
+method to access to its associated profile::
+
+    $profile = $container->get('profiler')->loadProfileFromResponse($response);
+
+When the profiler stores data about a request, it also associates a token with it;
+this token is available in the ``X-Debug-Token`` HTTP header of the response.
+Using this token, you can access the profile of any past response thanks to the
+:method:`Symfony\\Component\\HttpKernel\\Profiler\\Profiler::loadProfile` method::
+
+    $token = $request->headers->get(X-Debug-Token);
+    $profile = $container->get('profiler')->loadProfile($token);
+
+.. tip::
+
+    When the profiler is enabled but not the web debug toolbar, use a tool like
+    Firebug to get the value of the ``X-Debug-Token`` HTTP header.
```

Comment:
all browsers have built in tools to do this instead of Firebug. I'm not sure if we want to promote plugin usage here.